docs/get-started/installation/microcontrollers/development-setup.md

```diff
@@ -220,18 +220,13 @@ Navigate to [the Viam app](https://app.viam.com) and [add a new machine](/cloud/
 Click on the name of the machine to go to the machine's page.
 Then, navigate to the **CONFIGURE** tab.
 
-### Configure your machine with an ESP32
-
 [Client API](/appendix/apis/) usage with the micro-RDK is currently limited to the following supported {{< glossary_tooltip term_id="resource" text="resources" >}}:
 
-- [Base](/build/micro-rdk/base/)
-- [Board](/build/micro-rdk/board/)
-- [Encoder](/build/micro-rdk/encoder/)
-- [Movement Sensor](/build/micro-rdk/movement-sensor/)
-- [Motor](/build/micro-rdk/motor/)
-
-See [micro-RDK](/build/micro-rdk/) to get a list of supported models and instructions on how to configure them.
-Follow [this guide](/build/micro-rdk/board/esp32/) to configure an `esp32` board on your machine.
+- [Base](/components/base/)
+- [Board](/components/board/)
+- [Encoder](/components/encoder/)
+- [Movement Sensor](/components/movement-sensor/)
+- [Motor](/components/motor/)
```

Comment:
[optional] could also remove this - I find it a bit odd to mention the Client API here in the middle of setup steps. I know we're on the Configure tab at this point but we aren't actually adding any components or services.